editoast/src/views/test_app.rs

```diff
@@ -106,15 +108,12 @@ impl TestAppBuilder {
         };
 
         // Setup tracing
-        let sub = tracing_subscriber::fmt()
-            .pretty()
-            .with_env_filter(
-                tracing_subscriber::EnvFilter::builder()
-                    .with_default_directive(tracing_subscriber::filter::LevelFilter::DEBUG.into())
-                    .from_env_lossy(),
-            )
-            .with_span_events(tracing_subscriber::fmt::format::FmtSpan::CLOSE)
-            .finish();
+        let exporter = opentelemetry_stdout::SpanExporter::default();
```

Comment:
As mentioned in some previous comments, I don't think we need `opentelemetry_stdout` since we already have `tracing_subscriber::layer::fmt()` which will take care of printing to the terminal. I would instead suggest using the [`NoopSpanExporter`](https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/testing/trace/struct.NoopSpanExporter.html).

```suggestion
        let exporter = opentelemetry_sdk::testing::trace::NoopSpanExporter::default();
```

Note that you will need to add the `testing` feature. Please do not add it on the `opentelemetry_sdk` crate in the `[dependencies]`. Add a new line for `opentelemetry_sdk` in the `[dev-dependencies]` (we don't want the `testing` feature to leak in production code).

Reply:
Done.